include/boost/geometry/geometry.hpp

```diff
@@ -20,11 +20,11 @@
 #ifndef BOOST_GEOMETRY_GEOMETRY_HPP
 #define BOOST_GEOMETRY_GEOMETRY_HPP
 
+#if !defined(BOOST_GEOMETRY_DISABLE_DEPRECATED_03_WARNING)
 #include <boost/config.hpp>
+#if defined(BOOST_NO_CXX14_CONSTEXPR) || (__cplusplus < 201402L)
```

Comment:
This will evaluate to true on all versions of MSVC by default (you have to set a special compile switch for MSVC to define a "orrect" value for `__cplusplus`). I thought the conclusion in the other thread was to just check for `BOOST_NO_CXX14_CONSTEXPR`?

Reply:
I meant to use `_MSVC_LANG` in stead of  `__cplusplus`. I'll remove that sub-condition.

> I thought the conclusion in the other thread was to just check for BOOST_NO_CXX14_CONSTEXPR?

Yes, that was the plan. I'll update it.